src/main/java/org/isf/patient/rest/PatientController.java

```diff
@@ -1,9 +1,8 @@
 package org.isf.patient.rest;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.*;
```

Comment:
Project standards to not allow package (wildcard) imports; all imports must be enumerated.